.editorconfig

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
Just starting to review, but do you mind getting rid of this file?

Reply:
Yeah I can remove it if you'd like - just thought it would be helpful and was a fairly standard thing.